101-sql-database-Import/README.md

```diff
@@ -0,0 +1,20 @@
+# Azure SQL Database
+
+![Azure Public Test Date](https://azurequickstartsservice.blob.core.windows.net/badges/101-sql-database-import/PublicLastTestDate.svg)
```

Comment:
check the pipeline results for changes that need to be made here (casing issue in the url)

Reply:
Done